xml/System.Reflection.PortableExecutable/PEDirectoriesBuilder.xml

```diff
@@ -60,8 +60,8 @@
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>The entry point address.</summary>
```

Comment:
@tmat This could also get a better description.

Reply:
Actually I found a good one here: https://github.com/dotnet/corefx/blob/d3911035f2ba3eb5c44310342cc1d654e42aa316/src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEHeader.cs

```suggestion
        <summary>The address of the entry point relative to the image base when the PE file is loaded into memory.</summary>
```